src/parsers/manifestjson.js

```diff
@@ -47,6 +47,17 @@ import {
 } from 'utils';
 import BLOCKED_CONTENT_SCRIPT_HOSTS from 'blocked_content_script_hosts.txt';
 
+/**
+ * @typedef {Object} Metadata
+ * @property {string} id
+ * @property {number} manifestVersion
+ * @property {string} name
+ * @property {number} type
+ * @property {string} version
+ * @property {string} firefoxMinVersion
+ * @property {Set<string>} experimentApiPaths
+ */
```

Comment:
I moved it down. I added JSDoc, because it helped me a great deal understand what structure I can rely on. I thought it might help future devs aswell, that do not know the internals yet.

Reply:
I am not against but I think I'd rather see that added in a different PR.